### PR TITLE
Update/arbutus wip

### PR DESCRIPTION
**Pull request by alaczek**
* Added theme screenshot.
* Removed links from the footer social menu placeholder.

### CODE REVIEW

File: arbutus/block-template-parts/footer.html [hunk 1]
```diff
@@ -9,11 +9,7 @@
 <!-- /wp:paragraph -->
 
 <!-- wp:social-links {"iconColor":"primary","iconColorValue":"#343e47","size":"has-small-icon-size","className":"is-style-logos-only"} -->
-<ul class="wp-block-social-links has-small-icon-size has-icon-color is-style-logos-only"><!-- wp:social-link {"url":"#","service":"twitter"} /-->
-
-<!-- wp:social-link {"url":"#","service":"instagram"} /-->
-
-<!-- wp:social-link {"url":"#","service":"tumblr"} /--></ul>
+<ul class="wp-block-social-links has-small-icon-size has-icon-color is-style-logos-only"></ul>
```
Comment (scruffian) [inline arbutus/block-template-parts/footer.html]: You might want to make this a navigation block that implements a social nav. You can do that like this:

```<!-- wp:navigation {"itemsJustification":"center","overlayMenu":"never","__unstableSocialLinks":"social"} /-->```

Comment (alaczek) [reply arbutus/block-template-parts/footer.html]: Hmm... I do want a specific look for the social links nav in the footer (icons only, no bg, custom color, custom icon size), and I can't seem to achieve this inside a nav menu. Am I missing something?
image

Comment (scruffian) [reply arbutus/block-template-parts/footer.html]: We need [this PR](https://github.com/Automattic/themes/pull/4984) to be merged first...

Comment (MaggieCabrera) [reply arbutus/block-template-parts/footer.html]: that PR will only help with the color, not the custom size, I believe

Comment (scruffian) [reply arbutus/block-template-parts/footer.html]: we could do custom size with CSS for now..

Comment (alaczek) [reply arbutus/block-template-parts/footer.html]: I used your simplified setup, and I think it's good to leave like that.

**Comment by alaczek** [comment]
I made the changes requested, so I'm going to go ahead and merge.

**Pull request merged**